src/components/pages/books/Books.jsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import './books.css';
+import AddNewBook from './addNewBook';
+
+const Books = () => (
```

Comment:
- [ ] You could refactor your Books component to render a list of books using a <Book> component. This would make your code more reusable and maintainable. The <Book> component should take the book data as a prop and render the title and the author. For example:

```
const Books = () => {
  // You can use an array of objects to store your book data
  const books = [
    {
      genre: "Action",
      title: "The Hunger Games",
      author: "Suzanne Collins",
      chapter: "Chapter 17"
    },
    {
      genre: "Science Fiction",
      title: "Dune",
      author: "Frank Herbert",
      chapter: "Chapter 3: A lesson Learned"
    }
  ];

  return (
    <>
      <div className="book">
        <div className="book-section">
          // You can use the map method to iterate over the books array and render a <Book> component for each book
          {books.map((book) => (
            <Book key={book.title} book={book} />
          ))}
        </div>
        <AddNewBook />
      </div>
    </>
  );
};

// The <Book> component should take the book prop and render the title and the author
const Book = ({ book }) => {
  return (
    <div className="book-section-one">
      <div className="action-section">
        <span>{book.genre}</span>
        <h2>{book.title}</h2>
        <p>{book.author}</p>
        <div className="book-changes">
          <button type="button" id="comment">
            Comment
          </button>
          <div className="vh" />
          <button type="button" id="remove">
            Remove
          </button>
          <div className="vh" />
          <button type="button" id="edit">
            Edit
          </button>
        </div>
      </div>
      <div className="vh2" />

      <div className="update-progress">
        <span>CURRENT CHAPTER</span>
        <h4>{book.chapter}</h4>

        <button type="button">UPDATE PROGRESS</button>
      </div>
    </div>
  );
};
```

Reply:
Done.